src/tribler/gui/tribler_window.py

```diff
@@ -240,11 +241,19 @@ def __init__(
 
         self.debug_pane_shortcut = QShortcut(QKeySequence("Ctrl+d"), self)
         connect(self.debug_pane_shortcut.activated, self.clicked_debug_panel_button)
+
         self.import_torrent_shortcut = QShortcut(QKeySequence("Ctrl+o"), self)
         connect(self.import_torrent_shortcut.activated, self.on_add_torrent_browse_file)
+
         self.add_torrent_url_shortcut = QShortcut(QKeySequence("Ctrl+i"), self)
         connect(self.add_torrent_url_shortcut.activated, self.on_add_torrent_from_url)
 
+        self.tribler_gui_test_exception_shortcut = QShortcut(QKeySequence("Ctrl+Alt+Shift+Home"), self)
+        connect(self.tribler_gui_test_exception_shortcut.activated, self.on_test_tribler_gui_exception)
+
+        self.tribler_core_test_exception_shortcut = QShortcut(QKeySequence("Ctrl+Alt+Shift+End"), self)
+        connect(self.tribler_core_test_exception_shortcut.activated, self.on_test_tribler_core_exception)
+
```

Comment:
I don't have 'Home' and 'End' keys on my keyboard. What about changing it to C (core) and G (GUI)?

So, my proposition is to change them to:
1. Ctrl+Alt+Shift+G
2. Ctrl+Alt+Shift+C